diesel/src/expression/mod.rs

```diff
@@ -1,14 +1,14 @@
 //! AST types representing various typed SQL expressions. Almost all types
-//! implement either [`Expression`](trait.Expression.html) or
-//! [`AsExpression`](trait.AsExpression.html).
+//! implement either [`Expression`](/diesel/expression/trait.Expression.html) or
```

Comment:
Can you change this to avoid absolute URLs? This will eventually break when we start hosting past versions.